config/config.example.yml

```diff
@@ -207,6 +207,13 @@ item:
     undoTimeout: 10000 # 10 seconds
   # Show the item access status label in items lists
   showAccessStatuses: false
+  simpleView:
+    bitstreamPageSize: 5
+  fullView:
+    # Number of entries in the bitstream list in the full item view page.
+    # Rounded to the nearest size in the list of selectable sizes on the
+    # settings menu.  See pageSizeOptions in 'pagination-component-options.model.ts'.
+    bitstreamPageSize: 5
```

Comment:
Do you have a clear use case as to why you'd want this set differently for `simpleView` vs `fullView`?   I was expecting this to just be a single setting used on both pages, as I don't think a site should need to display the `fullView` that much differently than the `simpleView`.

I'd also prefer this setting be named `pageSize` so that it's identical to all other `pageSize` configurations.  So, I'd prefer if we simplified this to...
```
# If we have just one setting for both views
item:
  bitstream:
    pageSize: 5
```

If you really feel we need this to be different for different Item views, then we could do something like..
```
item:
  simpleView:
    bitstream:
      pageSize: 5
  fullView:
    bitstream:
      pageSize: 5
```
But, again, I don't see a clear use case for having this be customized differently for different views.  It seems like most sites will want this to be the same.

Reply:
Hi @tdonohue 

> @arvoConsultores : Do you have a clear use case as to why you'd want this set differently for `simpleView` vs `fullView`?

No, there's no specific use case, i've divided this in two because "simple" and "full" are two separate pages in the code. No problem in change it to just one property.

> I'd also prefer this setting be named `pageSize` so that it's identical to all other `pageSize` configurations.

Yes, that's true, it's more clear, i will change it to item > bitstream > pageSize